Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+--
+
+ALTER TABLE ONLY public."measuredFragments"
+    ADD CONSTRAINT "measuredFragments_pkey" PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2994 (class 2606 OID 18106)
+-- Name: measuredPools measuredPools_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."measuredPools"
+    ADD CONSTRAINT "measuredPools_pkey" PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2930 (class 2606 OID 18151)
+-- Name: modelMetabolites modelMetabolites_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelMetabolites"
+    ADD CONSTRAINT "modelMetabolites_pkey" PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2932 (class 2606 OID 18164)
+-- Name: modelPathways modelPathways_id_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelPathways"
+    ADD CONSTRAINT "modelPathways_id_key" UNIQUE (id);
+
+
+--
+-- TOC entry 2934 (class 2606 OID 18166)
+-- Name: modelPathways modelPathways_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelPathways"
+    ADD CONSTRAINT "modelPathways_pkey" PRIMARY KEY (model_id, pathway_id);
+
+
+--
+-- TOC entry 2936 (class 2606 OID 18153)
+-- Name: modelReactions modelReactions_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelReactions"
+    ADD CONSTRAINT "modelReactions_pkey" PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2996 (class 2606 OID 18084)
+-- Name: models models_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.models
+    ADD CONSTRAINT models_pkey PRIMARY KEY ("id ");
+
+
+--
+-- TOC entry 3000 (class 2606 OID 18155)
+-- Name: simulationParameters simulationParameters_ simulation_id_simulation_dateAndTime_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."simulationParameters"
+    ADD CONSTRAINT "simulationParameters_ simulation_id_simulation_dateAndTime_key" UNIQUE (" simulation_id", "simulation_dateAndTime");
+
+
+--
+-- TOC entry 3002 (class 2606 OID 18017)
+-- Name: simulationParameters simulationParameters_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."simulationParameters"
+    ADD CONSTRAINT "simulationParameters_pkey" PRIMARY KEY ("id ");
+
+
+--
+-- TOC entry 2998 (class 2606 OID 18095)
+-- Name: simulation simulation_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.simulation
+    ADD CONSTRAINT simulation_pkey PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2914 (class 2606 OID 18157)
+-- Name: tracers tracers_met_id_met_name_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tracers
+    ADD CONSTRAINT tracers_met_id_met_name_key UNIQUE (met_id, met_name);
+
+
+--
+-- TOC entry 2916 (class 2606 OID 18159)
+-- Name: tracers tracers_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tracers
```

Comment:
We are missing the foreign keys for these MFA tables and their association with the other ISATab tables.  This is something that will need to be added that was not included in the table information I had sent you previously.  Let's discuss how best to update these in another issue.